sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 2.9.0-beta.6 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+- Fixes the potential NullPointerException in ReactorSession if the thread constructing ReactorSession ever happens to run the disposeWork (cleanup phase) synchronously. ([36916](https://github.com/Azure/azure-sdk-for-java/issues/36916))
```

Comment:
nit; this entry should be removed as it is done and belonged to 2.8.10

Reply:
Won't this be released in that beta as well though?